ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/custom-endpoints/disconnected/cucushift-installer-rehearse-gcp-ipi-custom-endpoints-disconnected-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-custom-endpoints-disconnected
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-custom-endpoints-disconnected-provision
```

Comment:
Per https://github.com/openshift/release/pull/67158, do you mind to append `- ref: cucushift-installer-reportportal-marker` here ?